{"name": "test_create_token", "status": "failed", "statusDetails": {"message": "AssertionError: The sataus code is wrong", "trace": "self = <tests.test.crud.test_crud.TestCrud object at 0x0000015E04580340>\n\n    def test_create_token(self):\n        response=post_request(url=APIConstants.url_create_token(),\n                              auth=None,\n                              headers=Util().common_header_json(),\n                              payload=payload_create_token(),in_json=False)\n    \n>       verify_http_status_code(response,200)\n\ntest_crud.py:23: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nresponse_data = <Response [200]>, expected_data = 200\n\n    def verify_http_status_code(response_data, expected_data):\n>       assert response_data == expected_data ,\"The sataus code is wrong\"\nE       AssertionError: The sataus code is wrong\n\n..\\..\\..\\src\\helpers\\common_verification.py:2: AssertionError"}, "start": 1730100525181, "stop": 1730100526430, "uuid": "796c3fe7-dd29-4fd6-96cf-0e2505f31fee", "historyId": "7473351ba3825807ea1c6056c6f3fded", "testCaseId": "30d2bb341fcd2fcbd16542e829364bef", "fullName": "tests.test.crud.test_crud.TestCrud#test_create_token", "labels": [{"name": "parentSuite", "value": "tests.test.crud"}, {"name": "suite", "value": "test_crud"}, {"name": "subSuite", "value": "TestCrud"}, {"name": "host", "value": "LAPTOP-J82A4UMN"}, {"name": "thread", "value": "16188-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test.crud.test_crud"}]}